Created GoalDeleteConfirmationModal

sveinarneroed opened the pull request.

Commit:
Made GoalDeleteConfirmationModal

tomhah committed:
Merge branch 'master' into SveinArne

Commit:
Final edit

default text

nathan-bird commented on src/GoalDeleteConfirmationModal/index.js:
```diff
@@ -0,0 +1,44 @@
+      <SectionFooterCancelOK
+        onCancel={props.onCancel}
+        onClick={props.onOk}
+        okButtonText="SLETT"
```
place 'okButtonText' prop first to get them alphabetical.

nathan-bird commented on src/GoalDeleteConfirmationModal/index.js:
```diff
@@ -0,0 +1,44 @@
+        <Typography type="bodyTextNormal">
+          {props.children}
+        </Typography>
+
```
unneeded linebreak

nathan-bird commented on src/GoalDeleteConfirmationModal/stories.js:
```diff
@@ -0,0 +1,11 @@
+import {stories, className, json, number, bool, string} from '../duckyStories';
```
json and number aren't used

nathan-bird commented on src/GoalDeleteConfirmationModal/index.js:
```diff
@@ -0,0 +1,44 @@
+  return (
+    <Modal
+      onHide={props.onCancel}
+      show
```
a show prop is needed here unless the modal will always be open. also add it to props validation

nathan-bird commented on src/GoalDeleteConfirmationModal/stories.js:
```diff
@@ -0,0 +1,11 @@
+  title: string("Tittel"),
+  children: string("Default tekst"),
+  show: bool()
+});
```
add all props to the story so it is clear what the component takes, eg. onOk, onCancel etc

nathan-bird reviewed (changes requested):
Nice component.
A few style comments and unused variables.
There a couple of points that need some clarification though.

nathan-bird commented on src/GoalDeleteConfirmationModal/index.js:
```diff
@@ -0,0 +1,44 @@
+  onCancel: PropTypes.func,
+  onClick: PropTypes.func,
+  onOk: PropTypes.func,
+  text: PropTypes.string,
```
text prop doesn't exist

nathan-bird commented on src/GoalDeleteConfirmationModal/stories.js:
```diff
@@ -0,0 +1,11 @@
+stories(module, GoalDeleteConfirmationModal, [
+  "https://github.com/DuckyTeam/ducky-web/issues/1442"
+], {
+  className: className(),
```
dontt have a className prop on the component

nathan-bird commented on src/GoalDeleteConfirmationModal/index.js:
```diff
@@ -0,0 +1,44 @@
+GoalDeleteConfirmationModal.propTypes = {
+  children: PropTypes.node,
+  onCancel: PropTypes.func,
+  onClick: PropTypes.func,
```
onClick prop doesn't exist

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into SveinArne

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into SveinArne

Commit:
Created LabelNumberDisplay2

NB: Tooltip not ok

Commit:
Fixed GoalDeleteConfirmationModal+Created LabelNumberDisplay2

Fixed GoalDeleteConfirmationModal according to comments from Nathan:
https://github.com/DuckyTeam/ducky-components/pull/273

Commit:
fixed GoalDelteConfirmationModal+Created LabelNumberDisplay2
